Containers/talk/start.sh

```diff
@@ -55,12 +55,10 @@ TURN_CONF
 set -x
 sed -i 's|#interface.*|interface = "lo"|g' /etc/janus/janus.transport.websockets.jcfg
```

Comment:
this option does not exist in this file @szaimen

Reply:
I am not sure. This was copied from somewhere else.

Reply:
but as we don’t expose janus ports, we could leave it as it is?

Reply:
I suppose so, yes

Reply:
I remove it

Reply:
Okay